Nothing better to do: fix a small warning in mxc-wdt

  CC      drivers/watchdog/mxc_wdt.o                                                                                                                                           
drivers/watchdog/mxc_wdt.c: In function 'mxc_wdt_shutdown':                                                                                                                    
drivers/watchdog/mxc_wdt.c:299: warning: unused variable 'res' 

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 drivers/watchdog/mxc_wdt.c |    1 -
 1 file changed, 1 deletion(-)

Index: linux-2.6.26/drivers/watchdog/mxc_wdt.c
===================================================================
--- linux-2.6.26.orig/drivers/watchdog/mxc_wdt.c	2009-07-26 15:42:30.000000000 +0200
+++ linux-2.6.26/drivers/watchdog/mxc_wdt.c	2009-07-26 15:42:43.000000000 +0200
@@ -296,7 +296,6 @@ static int __init mxc_wdt_probe(struct p
 
 static void mxc_wdt_shutdown(struct platform_device *pdev)
 {
-	struct resource *res = platform_get_drvdata(pdev);
 	mxc_wdt_disable(wdt_base_reg);
 	pr_info("MXC Watchdog # %d shutdown\n", dev_num);
 }
